src/lib/eliom_common.server.ml

```diff
@@ -1270,24 +1270,39 @@ let create_persistent_table name =
   perstables := Perstables.add name !perstables;
   Ocsipersist.open_table name
 
-let persistent_cookies_table :
-    (full_state_name * float option * timeout * perssessgrp option)
-    Ocsipersist.table Lwt.t Lazy.t =
-  lazy (create_persistent_table eliom_persistent_cookie_table)
```

Comment:
Do we still need function create_persistent_table?

Reply:
It's utilised by `Eliom_state.create_persistent_table`. I'd prefer not to introduce breaking changes in this PR. Maybe later I'll port the other persistent tables to the functorial interface as well and get rid of some legacy code.